apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -18,7 +18,7 @@ import {AnimationProps} from '@cdo/apps/p5lab/shapes';
 import {isMobileDevice} from '@cdo/apps/util/browser-detector';
 import {PICKER_TYPE} from './AnimationPicker.jsx';
 import style from './animation-picker-body.module.scss';
-import AnimationUploadButton from './AnimationUploadButton.jsx';
+import {UnconnectedAnimationUploadButton as AnimationUploadButton} from './AnimationUploadButton.jsx';
```

Comment:
We should still use the connected component here right?

Reply:
Yes - definitely. Thanks!